### DISCUSSION TITLE
breeze theme - PDP gallery slidier navigation arrow with dots, with one and half image at a time

### DISCUSSION THREAD

**Discussion by Maitreya11111** [breezefront/community#53] [1. Help]
I am using the breeze theme. there is a different UI for navigation arrows. I want to remove thumbnail images and instead this. I want to show dots with arrows and one and a half images at a time in the gallery. I tried different ways but was not able to get results.
for reference, I am attaching both images.
one is for the current PDP slider and the other one is for the expected PDP slider.

current slider
![currrent](https://user-images.githubusercontent.com/122878788/213412345-7f754cdc-d7bc-461b-bb2c-64cb173d5cc3.png)

expected slider
![expected-slider](https://user-images.githubusercontent.com/122878788/213412353-7a485b60-0155-4474-bf65-3eeeff75113a.png)

**Comment by vovayatsyuk** [comment]
1. You can add custom styles to transform thumb images into dots. Something like that:

    ```
    .thumbnails img { display: none; }
    .thumbnails li { width: 12px; height: 2px; background: #000; }
    ```

2. Unfortunately breeze gallery doesn't support multiple images at once.
